resources/views/maps/markers/create.blade.php

```diff
@@ -42,39 +42,9 @@
         <div class="modal-footer">
 
             <div class="pull-left">
-                @include('partials.footer_cancel')
-            </div>
-            <div class="submit-group">
-                <input id="submit-mode" type="hidden" value="true"/>
-                <div class="btn-group">
-                    <button class="btn btn-success" id="form-submit-main"
-                        data-target="{{ isset($target) ? $target : null }}">{{ __('crud.save') }}</button>
-                    <button type="button" class="btn btn-success dropdown-toggle" data-toggle="dropdown"
-                        aria-expanded="false">
-                        <span class="caret"></span>
-                    </button>
-                    <ul class="dropdown-menu dropdown-menu-right" role="menu">
-                        <li>
-                            <a href="#" class="dropdown-item form-submit-actions">
-                                {{ __('crud.save') }}
-                            </a>
-                        </li>
-                        <li>
-                            <a href="#" class="dropdown-item form-submit-actions" data-action="submit-update">
-                                {{ __('crud.save_and_update') }}
-                            </a>
-                        </li>
-                        <li>
-                            <a href="#" class="dropdown-item form-submit-actions" data-action="submit-explore">
-                                {{ __('maps/markers.actions.save_and_explore') }}
-                            </a>
-                        </li>
-                    </ul>
-                </div>
-            </div>
-            <div class="submit-animation" style="display: none;">
-                <button class="btn btn-success" disabled><i class="fa-solid fa-spinner fa-spin"></i></button>
+                @include('partials.footer_cancel', ['ajax' => null])
             </div>
+            @include('maps.markers._actions')
         </div>
     </div>
```

Comment:
Something is wrong in the HTML, it's creating this ugly line in the footer 
![image](https://user-images.githubusercontent.com/3263440/197284974-9acede4d-0f54-4436-a005-869714effabe.png)
See how I fixed it in Layers